.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,50 @@
+name: Nightly-build
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
```

Comment:
Can we make it event triggered? for example, only when we merge a commit?

Reply:
Do you mean we run nightly build on event-based(commit to main branch)?